src/PDS4_CHAN1_IngestLDD.xml

```diff
@@ -1,18 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1E00.sch"
+   schematypens="http://purl.oclc.org/dsdl/schematron"?>
 
-<?xml-model href="https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1D00.sch"
-    schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<Ingest_LDD
+		xmlns="http://pds.nasa.gov/pds4/pds/v1"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation=" http://pds.nasa.gov/pds4/pds/v1  http://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1E00.xsd">
 
-<!-- 
-    02/07/2020 GEO/EAG Initial version of the Chandrayaan mission dictionary for PDS3 to PDS4 migration
-
--->
-
-<Ingest_LDD xmlns="http://pds.nasa.gov/pds4/pds/v1"
- xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
- xsi:schemaLocation="http://pds.nasa.gov/pds4/pds/v1 https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1D00.xsd">
     <name>Chandrayaan-1 Mission Dictionary</name>
-    <ldd_version_id>1.0.0.0</ldd_version_id>
+    <ldd_version_id>1.1.0.0</ldd_version_id>
```

Comment:
Looks like we need to push the version to 1.1.1.0 (with your potential updates).

Reply:
Will change.